### DISCUSSION TITLE
Erinnerungen

### DISCUSSION THREAD

**Discussion by benedictweis** [SE-TINF22B2/G5-DuoGradus#8] [Anforderungen]
Als Grundelement soll die Anwendung Erinnerungen an den Nutzer senden bei verschiedenen Events. Diese Erinnerungen sollen zunächst per Email gesendet werden.

**Comment by Jstn2004** [comment]
vielleicht auch beim erstmaligen anmelden abfragen, ob man überhaupt Benachrichtigungen bekommen möchte.
